### PR TITLE
Gettingstartedtutorialtwo

### PR DESCRIPTION
**Pull request by BillWagner**
This is the second console tutorial.

It demonstrates reading from the web, and more C# object oriented practices.

### CODE REVIEW

File: docs/tutorials/getting-started-with-csharp/console-webapiclient.md [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+method to block and wait for the task to finish:
+
+```cs
+public static void Main(string[] args)
+{
+    ProcessRepositories().Wait();
+}
+```
+
+Now, you have a program that does nothing, but does it asynchronously. Let's go back to the
```
Comment (ghogen) [inline docs/tutorials/getting-started-with-csharp/console-webapiclient.md]: :-)

File: docs/tutorials/getting-started-with-csharp/console-webapiclient.md [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+This change means you need to change the code that writes the name of each repository in program.cs:
+
+```cs
+Console.WriteLine(repo.Name);
+```
+
+Do a "dotnet build", followed by a "dotnet run" to make sure you've got the mappings correct. You should
+see the same output as before. Before we process more properties from the web server, let's make one
+more change to the `Repository` class. The `Name` member is a publicly accessible field. That's not
+a good object oriented practice, so let's hcange it to a property. For our purposes, we don't need
```
Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-webapiclient.md]: typo: hcange

File: docs/tutorials/getting-started-with-csharp/console-webapiclient.md [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+of all the project dependencies. You do not need to put the file in source
+control; it will be generated when you run “dotnet restore”. 
+
+After restoring packages, you run “dotnet build”. This executes the build
+engine and creates your application. Finally, you execute “dotnet run” to
+run your application.
+
+# Adding New Dependencies
+One of the key design goals for .NET Core is to minimize the size of
+the .NET framewwork installation. The .NET Core Standard library contains
```
Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-webapiclient.md]: typo: framewwork

File: docs/tutorials/getting-started-with-csharp/console-webapiclient.md [hunk 4]
```diff
@@ -0,0 +1,475 @@
...
+After you've made these changes, you should run "dotnet restore" again so
+that those packages are installed on your system.
+
+# Making Web Requests
+Now you're ready to start retrieving data from the web. In this
+application, you'll read information from the 
+[GitHub API](https://developer.github.com/v3/). Let's read information
+about the projects under the
+[.NET Foundation](http://www.dotnetfoundation.org/) umbrella. You'll
+start by making the request to the Github API to retrieve information
```
Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-webapiclient.md]: shouldn't it be GitHub with capital H?

File: docs/tutorials/getting-started-with-csharp/console-webapiclient.md [hunk 5]
```diff
@@ -0,0 +1,475 @@
...
+[DataMember(Name="homepage")]
+public Uri Homepage { get; set; }
+
+[DataMember(Name="watchers")]
+public int Watchers { get; set; }
+```
+
+These properties have built in conversions from the string type (which is what the JSON packets contain) to
+the target type. The `Uri` type may be new to you. It represents a URI, or in this case, a URL. In the case
+of the `Uri` and `int` types, if the JSON packet contains data that does nto convert to the target type,
```
Comment (mairaw) [inline docs/tutorials/getting-started-with-csharp/console-webapiclient.md]: typo: nto

**Pull request merged**